test/e2e-test/pages/common/file-upload-common.ts

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
I would move this file outside of common folder and maybe create a folder for file-upload